doc/source/NewSiteConfigs.rst

```diff
@@ -203,10 +203,10 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    cd envs/unified-env.mymacos/
    spack env activate [-p] .
 
-3. Temporarily set environment variable ``SPACK_SYSTEM_CONFIG_PATH`` to modify site config files in ``envs/unified-env.mymacos/site``
+3. Change directory back to the top level of the cloned spack-stack repository and temporarily set environment variable ``SPACK_SYSTEM_CONFIG_PATH`` to modify site config files in ``envs/unified-env.mymacos/site``
 
 .. code-block:: console
-
+   cd ../../
```

Comment:
If you do that, you'll have to add instructions to cd back into the env dir. Would it be better to change item 3 above?
```
Still in the environment directory, temporarily set environment variable ``SPACK_SYSTEM_CONFIG_PATH`` to modify site config files in ``site``:

export SPACK_SYSTEM_CONFIG_PATH="$PWD/site"
```
?

Reply:
yes, that seems better